index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang = "en">
+    <head>
+        <title>
+            Apple Store
+        </title>
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+            <meta charset="UTF-8">
+            <link rel="stylesheet" type="text/css" href="stylesheets/style.css">
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+            <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100&display=swap" rel="stylesheet">
+            <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300&display=swap" rel="stylesheet">
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        </head>
+    <body class = "flex flex-column">
+        <div class="background">
+            <header>
+                <nav>
+                    <ul>
+                        <li class = "border-l"><a href="#"><span class="center logo"><i class="fa fa-apple" aria-hidden="true"></i></span></a></li>
+                        <li><a href="#"><span class="center">Store</span></a></li>
+                        <li><a href="#"><span class="center">Mac</span></a></li>
+                        <li><a href="#"><span class="center">ipod </span></a></li>
+                        <li><a href="#"><span class="center">iphone</span></a></li>
+                        <li><a href="#"><span class="center">ipad</span></a></li>
+                        <li><a href="#"><span class="center">itunes</span></a></li>
+                        <li><a href="#"><span class="center">Support</span></a></li>
+                        <li class = "last">
+                                <span class="container-search">
+                                    <input type = "text" class = "input-search" />
+                                    <i class="fa fa-search icon-search"></i>
+                                </span>             
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+            <main class = "flex flex-column">
+                <section class="background-header">
+                    <div>
+                        <h1>What will your verse be</h1>
+                    </div>
+                    <div>
+                        <p>Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.
+                        </p>
+                    </div>
+    
+                    <div>
+                        <p>
+                            <a href = "#">Watch the “Your Verse” film and explore the stories</a> 
+                            <i class="fa fa-play-circle watch-icon"></i>
+                        </p>
+                    </div>
```

Comment:
- [ ] ```<h>```, and ```<p>``` are HTML block elements (not inline), using a div to wrap them is redundant and unnecessary.